visidata/modify.py

```diff
@@ -284,4 +284,4 @@ def new_rows(sheet, n):
 Sheet.addCommand('za', 'addcol-new', 'addColumnAtCursor(SettableColumn(input("column name: "))); cursorRight(1)', 'append an empty column')
 Sheet.addCommand('gza', 'addcol-bulk', 'addColumnAtCursor(*(SettableColumn() for c in range(int(input("add columns: "))))); cursorRight(1)', 'append N empty columns')
 
-Sheet.addCommand('z^S', 'commit-sheet', 'commit()', 'commit changes back to source.  not undoable!')
+Sheet.addCommand('z^S', 'commit-sheet', 'commit()', 'Commit changes back to data source. You cannot undo these changes.')
```

Comment:
How do we feel about having "You" in the helpstrings?  I think this would be the first usage, and we should make a decision about how technical vs. conversational the helpstrings (and other docs) are.

Reply:
We've previously discussed writing some guidelines - I will create a PR with a strawman set as a starting point.